orchestration/dagster_orchestration/hca_orchestration/pipelines/validate_ingress.py

```diff
@@ -0,0 +1,62 @@
+from dagster import ModeDefinition, pipeline, failure_hook, HookContext
+
+from dagster_utils.resources.google_storage import google_storage_client, mock_storage_client
+from dagster_utils.resources.data_repo.jade_data_repo import jade_data_repo_client, noop_data_repo_client
+from dagster_utils.resources.slack import console_slack_client, live_slack_client
+
+from hca_manage.validation import run
+from hca_orchestration.config import preconfigure_resource_for_mode
+from hca_orchestration.contrib.slack import key_value_slack_blocks
+
+prod_mode = ModeDefinition(
+    name="prod",
+    resource_defs={
+        "data_repo_client": preconfigure_resource_for_mode(jade_data_repo_client, "prod"),
+        "gcs": google_storage_client,
+        "slack": preconfigure_resource_for_mode(live_slack_client, "prod"),
+    }
+)
+
+dev_mode = ModeDefinition(
+    name="dev",
+    resource_defs={
+        "data_repo_client": preconfigure_resource_for_mode(jade_data_repo_client, "dev"),
+        "gcs": google_storage_client,
+        "slack": preconfigure_resource_for_mode(live_slack_client, "dev"),
+    }
+)
+
+local_mode = ModeDefinition(
+    name="local",
+    resource_defs={
+        "data_repo_client": preconfigure_resource_for_mode(jade_data_repo_client, "dev"),
+        "gcs": google_storage_client,
+        "slack": console_slack_client,
+    }
+)
+
+test_mode = ModeDefinition(
+    name="test",
+    resource_defs={
+        "data_repo_client": noop_data_repo_client,
+        "gcs": mock_storage_client,
+        "slack": console_slack_client,
+    }
+)
+
+
+@failure_hook(
+    required_resource_keys={'slack', 'dagit_config'}
+)
+def validation_failed_notification(context: HookContext) -> None:
+    kvs = {
+        "Dagit link": f'<{context.resources.dagit_config.run_url(context.run_id)}|View in Dagit>'
+    }
+    context.resources.slack.send_message(blocks=key_value_slack_blocks("Validation Ingress Failed", key_values=kvs))
+
+
+@pipeline(
+    mode_defs=[prod_mode, local_mode, test_mode, dev_mode]
+)
+def validate_ingress() -> None:
+    run()
```

Comment:
Good start! I think we might have to feed in the staging path to the run command.
`python hca_manage/validation.py -p gs://PATH` is how the validation tool is normally called manually.
I think we need to get the input_prefix or the path of the staging area from the context but I'm not sure of we can do that directly here in this pipeline or if we need to set up a validate_ingress solid first.